superset/connectors/base/models.py

```diff
@@ -439,9 +439,7 @@ def query(self, query_obj: QueryObjectDict) -> QueryResult:
         """
         raise NotImplementedError()
 
-    def values_for_column(
-        self, column_name: str, limit: int = 10000, contain_null: bool = True,
-    ) -> List[Any]:
+    def values_for_column(self, column_name: str, limit: int = 10000,) -> List[Any]:
```

Comment:
nit (I wonder if this has been fixed in a recent version of black?)
```suggestion
    def values_for_column(self, column_name: str, limit: int = 10000) -> List[Any]:
```

Reply:
It seems not. Manually fixed